packages/dds/tree/src/test/utils.ts

```diff
@@ -370,13 +370,17 @@ export class TestTreeProvider {
 	}
 }
 
+type SharedTreeWithConnectionStateSetter = SharedTree & {
+	setConnected: (connectionState: boolean) => void;
```

Comment:
```suggestion
	readonly setConnected: (connectionState: boolean) => void;
```
You'll have to use `Mutable<SharedTreeWithConnectionStateSetter>` below to set `setConnected`